scripts/validation/providers/schema.json

```diff
@@ -295,6 +296,10 @@
                     "type": "string",
                     "enum": ["HMAC-SHA1", "PLAINTEXT"]
                 },
+                "signature_protocol": {
+                    "type": "string",
+                    "enum": ["WSSE"]
+                },
```

Comment:
what about a signature top level attributes with a `protocol` signature?
```
auth_mode: SIGNATURE
signature: 
   protocol: WSSE
```
just in case we need to add another signature related attribute in the future

Reply:
I have updated this with the latest [commit](https://github.com/NangoHQ/nango/pull/2942/commits/b4d3dd0ffcd24f1188b251597107f09a4e90eb13)